infrasim/cli.py

```diff
@@ -299,6 +299,11 @@ def add_command_parsers(subparser):
 
 
 def command_handler():
+    py_version = sys.version_info
+    if not (py_version[0] is 2 and py_version[1] is 7):
+        print "Warning: please install Python 2.7 first."
+        return
+    
```

Comment:
@turtle-fly How to solve it from pip = =? You mean there would be a library to adjust grammar among different python versions?